### ISSUE TITLE
VGrive stops syncing and closes randomly while running. 

### ISSUE THREAD

**Issue by tern19** [bcedu/VGrive#48]
I'm running latest version of Vgrive (1.2.4-1) on Manjaro KDE (
KDE Plasma Version: 5.17.2
KDE Frameworks Version: 5.63.0
Qt Version: 5.13.1
Kernel Version: 4.19.79-1-MANJARO)

 Usually, Vgrive runs for about a few minutes while the application is open and then it just suddenly stops. I've tried running it via terminal and it ran for about 10 seconds. For the better part of it this is the output that I got:
```
(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_string_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_string_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_boolean_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_string_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_string_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_boolean_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_string_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_string_member: assertion 'node != NULL' failed

(com.github.bcedu.vgrive:23527): Json-CRITICAL **: 13:06:46.145: json_object_get_boolean_member: assertion 'node != NULL' failed
LOG: INFO: open booklet.pdf ignored
LOG: INFO: Booklet for st104.odt ignored
LOG: INFO: Booklet for st104.odt~ ignored
LOG: INFO: r-cheat-sheet-3.pdf ignored

(com.github.bcedu.vgrive:23527): GLib-GObject-CRITICAL **: 13:06:46.146: g_object_unref: assertion 'old_ref > 0' failed
LOG: INFO: clec15.pdf ignored
LOG: INFO: clec8.pdf ignored
LOG: INFO: clec18.pdf ignored
LOG: INFO: clec10.pdf ignored
LOG: INFO: clec14.pdf ignored
LOG: INFO: clec17.pdf ignored
LOG: INFO: clec19.pdf ignored
LOG: INFO: clec23.pdf ignored
LOG: INFO: clec7.pdf ignored
LOG: INFO: clec12.pdf ignored
LOG: INFO: clec13.pdf ignored
LOG: INFO: clec24.pdf ignored
LOG: INFO: clec9.pdf ignored
**
```
And then it finished off with this:

`Gtk:ERROR:../gtk/gtk/gtktextview.c:4451:gtk_text_view_validate_onscreen: assertion failed: (priv->onscreen_validated)
Bail out! Gtk:ERROR:../gtk/gtk/gtktextview.c:4451:gtk_text_view_validate_onscreen: assertion failed: (priv->onscreen_validated)
LOG: INFO: clec22.pdf ignored
Aborted (core dumped)
`

**Comment by bcedu** [comment]
Could you update to v1.3.0 to check if its solved? In the 1.2.4 I introduced a bug that made that the app ignored the pdf files. I think this could have been the problem that caused de core dumped.

**Comment by tern19** [comment]
I think it's working now so updating it to 1.3.0 fixed it.
